Cargo.toml

```diff
@@ -11,3 +11,9 @@ documentation = "https://docs.rs/nb"
 readme = "README.md"
 version = "1.0.0" # remember to update html_root_url
 edition = "2018"
+
+[features]
+"defmt-0.3" = ["defmt"]
```

Comment:
I think it's fine to bump it. It will be a 1.1 release anyway.

Reply:
Yeah, that's the reason I didn't include it.
This feature resolution (with `dep:`) was introduced in 1.60 - https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies so it is quite the jump I would say. Not sure if that's ok + we can also include `rust-version` key as it is available since 1.56

Reply:
I'm also fine with bumping.